### DISCUSSION TITLE
Not nullable return_policy_id

### DISCUSSION THREAD

**Discussion by baahar** [etsy/open-api#779] [Q&A] [answered]
Hi,

In the documentation it says that updateListing accepts the nullable parameter return_policy_id. I assume that is to break a previously made association with a return policy.

But when I try to send any of the following
return_policy_id = null
return_policy_id = 0
return_policy_id = 

the return_policy_id of the listing does not get reset. How can I sever the link to a return policy?

Best,
baa

**Comment by mjec** [comment, marked as answer]
Hi @baahar, it is not possible to unset the return policy of a listing once a return policy is set. You can only change it to another return policy. The reason this field is nullable is so you can update the listing without updating the return policy.
